### PR TITLE
Contributing: Adding information about the weekly IRC meeting

### PR DESCRIPTION
**Pull request by sfrisk**
Fixes gh-98

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ Chassis is an attempt at creating open standards designed for CSS libraries, Jav
 
 If you maintain a JavaScript library that has UI components, we'd love for you to join our efforts. We'd be happy to hear your feedback and work through updates of your project(s) to support CSS Framework. If you have the time and interest to help define the standards and improve on our implementation, that's even better! Check out the [contribution guidelines](https://github.com/jquery/css-chassis/blob/master/CONTRIBUTING.md) for more information.
 
+## Contributing
+
+For more information about contributing to Chassis, please (see our contributing guide)[CONTRIBUTING.md].
```
Comment (geekman-rohit) [inline README.md]: is see our contributing guide supposed to be hyperlink to CONTRIBUTING.md?  In that case the brackets got interchanged

Comment (sfrisk) [inline README.md]: Also I totally noticed there was a link right above it.  Getting rid of this entirely.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,3 +3,5 @@ Welcome! Thanks for your interest in contributing to Chassis. You're **almost**
 You may also want to take a look at our [commit & pull request guide](http://contribute.jquery.org/commits-and-pull-requests/) and [style guides](http://contribute.jquery.org/style-guide/) for instructions on how to maintain your fork and submit your code. Before we can merge any pull request, we'll also need you to sign our [contributor license agreement](http://contribute.jquery.org/cla).
 
 You can find us on [IRC](http://irc.jquery.org), specifically in #css-chassis should you have any questions. If you've never contributed to open source before, we've put together [a short guide with tips, tricks, and ideas on getting started](http://contribute.jquery.org/open-source/).
+
+We also have weekly meetings on Tuesday at 1pm ET on [IRC](http://irc.jquery.org) in #jquery-meeting. If you can't make the meeting, you can find the internal notes for each meeting on [the jQuery Meeting site](https://meetings.jquery.org/category/chassis/).
```
Comment (arschmitz) [inline CONTRIBUTING.md]: I might change this to just 
"Our weekly meetings are ..." i would also add a note that these are public meetings and all are welcome to attend and participate

**Comment by sfrisk** [comment]
I've updated this PR based on feedback. If everyone is okay with it by the end of today's meeting, I'll pull it in.

**Comment by arthurvr** [comment]
Shouldn't this be closed by https://github.com/jquery/css-chassis/commit/7e44a9639d0896d7a5ebbff38b611ba517539435? @sfrisk Don't forget the `closes` reference when merging PRs ;)

**Comment by sfrisk** [comment]
I could have sworn I wrote closed. bah.

**Pull request closed without merging**